types/react-big-calendar/lib/addons/dragAndDrop.d.ts

```diff
@@ -32,7 +32,7 @@ export interface withDragAndDropProps<TEvent extends object = Event, TResource e
     onDragStart?: ((args: OnDragStartArgs<TEvent>) => void) | undefined;
     onDragOver?: ((event: React.DragEvent) => void) | undefined;
     onDropFromOutside?: ((args: DragFromOutsideItemArgs) => void) | undefined;
-    dragFromOutsideItem?: (() => keyof TEvent | ((event: TEvent) => Date)) | undefined;
+    dragFromOutsideItem?: (() => TEvent) | undefined;
```

Comment:
It is being used without any parameters, as in
[`const event = this.context.draggable.dragFromOutsideItem()`](https://github.com/jquense/react-big-calendar/blob/master/src/addons/dragAndDrop/EventContainerWrapper.js#L139-L140)

or

[`const item = this.context.draggable.dragFromOutsideItem()`](https://github.com/jquense/react-big-calendar/blob/master/src/addons/dragAndDrop/WeekWrapper.js#L94-L95)